pkgs/by-name/mu/muon/package.nix

```
@@ -14,29 +16,47 @@
   embedSamurai ? false,
   buildDocs ? true,
 }:
-
 stdenv.mkDerivation (finalAttrs: {
   pname = "muon" + lib.optionalString embedSamurai "-embedded-samurai";
-  version = "0.4.0";
-
-  src = fetchFromSourcehut {
-    name = "muon-src";
-    owner = "~lattis";
-    repo = "muon";
-    tag = finalAttrs.version;
-    hash = "sha256-xTdyqK8t741raMhjjJBMbWnAorLMMdZ02TeMXK7O+Yw=";
-  };
+  version = "0.5.0";
+
+  srcs = [
+    (fetchFromSourcehut {
+      name = "muon-src";
+      owner = "~lattis";
+      repo = "muon";
+      tag = finalAttrs.version;
+      hash = "sha256-bWEYWUD+GK8R3yVnDTnzFWmm4KAuVPI+1yMfCXWcG/A=";
+    })
+    (fetchFromGitHub {
+      name = "meson-tests";
+      repo = "meson-tests";
+      owner = "muon-build";
+      rev = "452d9adde46edad8bf07ec2da69450a6fd2ef9fe";
+      hash = "sha256-CLG5m1yCeT7javRdXyDV5n5bRUvOktELZFHOMxPycfw=";
+    })
+  ]
+  ++ lib.optionals buildDocs [
+    (fetchFromGitHub {
+      name = "meson-docs";
+      repo = "meson-docs";
+      owner = "muon-build";
+      rev = "1017b3413601044fb41ad04977445e68a80e8181";
+      hash = "sha256-aFpyJFIqybLNKhm/kyfCjYylj7DE6muI1+OUh4Cq4WY=";
+    })
+  ];
+
+  sourceRoot = "./muon-src";
 
   outputs = [ "out" ] ++ lib.optionals buildDocs [ "man" ];
 
   nativeBuildInputs = [
     pkgconf
+    (python3.withPackages (ps: [ ps.pyyaml ]))
   ]
   ++ lib.optionals (!embedSamurai) [ samurai ]
-  ++ lib.optionals buildDocs [
-    (python3.withPackages (ps: [ ps.pyyaml ]))
-    scdoc
-  ];
+  ++ lib.optionals buildDocs [ scdoc ]
+  ++ lib.optionals stdenv.targetPlatform.isDarwin [ apple-sdk ];
```

`apple-sdk` is in darwin stdenv

```suggestion
  ++ lib.optionals buildDocs [ scdoc ];
```